test/functional/isVisible.js

```diff
@@ -1,9 +1,9 @@
-const { assert } = require('chai');
+const {assert} = require('chai');
 const firefox = require('selenium-webdriver/firefox');
 const webdriver = require('selenium-webdriver');
```

Comment:
You never actually use ``webdriver`` except to grab more imports out of it. Can you do it all in 1 line?